aws/test_utils.go

```diff
@@ -0,0 +1,19 @@
+package aws
+
+import (
+	"testing"
+
+	terratestaws "github.com/gruntwork-io/terratest/modules/aws"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/stretchr/testify/require"
+)
+
+func GetTestSession(t *testing.T, approvedRegions []string, forbiddenRegions []string) *session.Session {
```

Comment:
Hm, maybe I miss something, but the function `GetTestSession()` is not invoked in any place

Reply:
I modified the implementation a bit afterward. It's no longer being used. Will delete it.